src/Algebra/LabelledGraph/Theorems.agda

```diff
@@ -116,3 +156,53 @@ graph-laws {x = .(_ Graph.* (_ Graph.* _))} {.(_ Graph.* _ Graph.* _)} Graph.*as
 graph-laws {x = .(_ Graph.* (_ Graph.+ _))} {.(_ Graph.* _ Graph.+ _ Graph.* _)} Graph.left-distributivity = left-distributivity
 graph-laws {x = .((_ Graph.+ _) Graph.* _)} {.(_ Graph.* _ Graph.+ _ Graph.* _)} Graph.right-distributivity = right-distributivity
 graph-laws {x = .(_ Graph.* _ Graph.* _)} {.(_ Graph.* _ Graph.+ _ Graph.* _ Graph.+ _ Graph.* _)} Graph.decomposition = right-decomposition
+
+to-from-identity : ∀ {A} {x y : Graph A} -> x Graph.≡ y -> (to-graph (from-graph x)) Graph.≡ (to-graph (from-graph y))
```

Comment:
I don't think this proves that the composition of `to-graph` and `from-graph` is identity (as the name might imply). It should instead say: `∀ {A} {x : Graph A} -> (to-graph (from-graph x)) Graph.≡ x`. As it stands this definition seems pretty useless: it follows trivially from the combination of `graph-laws` and `labelled-graph-laws`.

Same for `from-to-identity`.